docs/contribution/50-reference/2-extensions.mdx

```diff
@@ -41,6 +42,21 @@ For more information about the individual attributes, see [which information an
         description: "Name of the extension",
         example: "Example Extension",
       },
+      subTitle: {
+        type: "object",
+        description: "A brief description of the extension",
```

Comment:
now we have a short description and a brief description. 

We should clarify the purpose in the description of the fields.